dynamic-plugins/wrappers/backstage-community-plugin-topology/package.json

```diff
@@ -1,34 +1,45 @@
 {
-  "name": "janus-idp-backstage-plugin-topology",
-  "version": "1.30.0",
+  "name": "backstage-community-plugin-topology",
+  "version": "1.29.4",
```

Comment:
This downgrade is right, but we need to increase the backstage version to 1.31 or something similar and check again that all updates are migrated!

* https://www.npmjs.com/package/@janus-idp/backstage-plugin-topology?activeTab=versions is currently 1.30.1
* https://www.npmjs.com/package/@backstage-community/plugin-topology?activeTab=versions is currently 1.29.4

Reply:
Hi @christoph-jerolimov , do you mean manually increase the @bacstage-community/plugin-topology version to 1.31? This part is handled by workflow, is it okay to manually change it?